.gitignore

```diff
@@ -115,3 +115,5 @@ recipes/ccr/ccr-target-hosts.json
 
 # Tracker tracks
 tracks/
```

Comment:
Nit: "Tracks" can be removed, and a comment can be added for the Visual Studio suffix like with the other entries.